### PR TITLE
Station Facelift

### PR DESCRIPTION
**Pull request by fernerr**
- Adds some new turf sprites to replace ones in common use across the station, ports the reinforced floor from bay.
 - Adds new vent, scrubber, sprites for many wall-mounted objects and holopad sprites that fit in better with the changed floors.
 - Remaps a ton of areas visually to make better use of these changed sprites.
UVwXeyPEOD
TaH8rglj8B
OljUmqelwG
RUvxMgxscN
lhQEkyLfcf

Thanks to Kyres1 and Happy_Fox for their fantastic spriting efforts.

- [x] Finish up widespread mapping tweaks.

Feedback thread: https://forums.aurorastation.org/topic/14038-station-facelift-feedback-thread/

### CODE REVIEW

**Comment by NonQueueingMatt** [comment]
i love this

**Comment by Geevies** [comment]
i love this

**Comment by ParadoxSpace** [comment]
i love this

**Comment by NotSchwann** [comment]
i love this

**Comment by TheDanseMacabre** [comment]
i love this

**Comment by alsoandanswer** [comment]
im not really sure about this

also arent these the same vents which failed to get merged last time

**Comment by fernerr** [comment]
Whilst I was hesitant to the new vents and scrubbers before, in the other pr, they clearly work very well in this new context, which the old ones would not.

**Comment by KingOfThePing** [comment]
This looks great, I love it. You are the hero we need, but not deserve.

**Comment by fernerr** [comment]
The spriters deserve the real credit, but thanks a lot for your support.
As for a feedback thread, I'll make one later today when I get a moment.

**Comment by fernerr** [comment]
Feedback thread: https://forums.aurorastation.org/topic/14038-station-facelift-feedback-thread/

**Comment by alsoandanswer** [comment]
Can we have pictures of the new walls?

**Comment by fernerr** [comment]
Yes, that's the plan.

**Comment by fernerr** [comment]
!review

File: code/__defines/color.dm [hunk 1]
```diff
@@ -27,6 +27,7 @@
 #define COLOR_YELLOW_GRAY      "#c9a344"
 #define COLOR_PALE_YELLOW      "#c1bb7a"
 #define COLOR_WARM_YELLOW      "#b3863c"
+#define COLOR_YELLOW_ALT       "#efbf2f"
```
Comment (NonQueueingMatt) [inline code/__defines/color.dm]: Bit undescriptive. Could use a better name.

File: code/game/objects/items/stacks/tiles/tile_types.dm [hunk 2]
```diff
@@ -24,8 +24,8 @@
  * Grass
  */
 /obj/item/stack/tile/grass
-	name = "grass tile"
-	singular_name = "grass floor tile"
+	name = "synthethic grass tile"
```
Comment (NonQueueingMatt) [inline code/game/objects/items/stacks/tiles/tile_types.dm]:
```suggestion
	name = "synthetic grass tile"
```

File: code/game/objects/items/stacks/tiles/tile_types.dm [hunk 3]
```diff
@@ -24,8 +24,8 @@
...
+	singular_name = "synthethic grass tile"
```
Comment (NonQueueingMatt) [inline code/game/objects/items/stacks/tiles/tile_types.dm]:
```suggestion
	singular_name = "synthetic grass tile"
```

File: code/game/objects/items/stacks/tiles/tile_types.dm [hunk 4]
```diff
@@ -24,8 +24,8 @@
  * Grass
  */
 /obj/item/stack/tile/grass
-	name = "grass tile"
-	singular_name = "grass floor tile"
+	name = "synthehic grass tile"
```
Comment (Geevies) [inline code/game/objects/items/stacks/tiles/tile_types.dm]:
```suggestion
	name = "synthetic grass tile"
```

File: code/game/objects/items/stacks/tiles/tile_types.dm [hunk 5]
```diff
@@ -24,8 +24,8 @@
...
+	singular_name = "synthehic grass tile"
```
Comment (Geevies) [inline code/game/objects/items/stacks/tiles/tile_types.dm]:
```suggestion
	singular_name = "synthetic grass tile"
```

File: code/game/turfs/flooring/flooring.dm [hunk 6]
```diff
@@ -40,8 +40,8 @@ var/list/flooring_types
 	var/footstep_sound = "tiles"
 
 /decl/flooring/grass
-	name = "grass"
-	desc = "Do they smoke grass out in space, Bowie? Or do they smoke AstroTurf?"
+	name = "synthethic grass"
```
Comment (Geevies) [inline code/game/turfs/flooring/flooring.dm]:
```suggestion
	name = "synthetic grass"
```

File: code/game/turfs/flooring/flooring.dm [hunk 7]
```diff
@@ -40,8 +40,8 @@ var/list/flooring_types
...
+	desc = "A patch of synthethic grass."
```
Comment (Geevies) [inline code/game/turfs/flooring/flooring.dm]:
```suggestion
	desc = "A patch of synthetic grass."
```

**Pull request merged**